### ISSUE TITLE
[Raspberry] Cannot install on Raspberry Pi OS

### ISSUE THREAD

**Issue by uzumakinaruto123** [electron/electron-rebuild#560]
Trying to install electron-rebuild to make serialport package work with electron but electron-rebuild installation throws the following error

![Screenshot 2020-12-27 203143](https://user-images.githubusercontent.com/15323421/103173747-09f68a80-4883-11eb-8a07-ff0cef84f395.jpg)

Tried many solutions like build from source, installation in different sequence, etc. no matter what this does not succeed.

![Screenshot 2020-12-27 203252](https://user-images.githubusercontent.com/15323421/103173751-11b62f00-4883-11eb-9898-e4381bd5125c.jpg)

**NPM log file**

[2020-12-27T15_00_08_402Z-debug.log](https://github.com/electron/electron-rebuild/files/5744696/2020-12-27T15_00_08_402Z-debug.log)

**Comment by aharish** [comment]
Yep, I am facing the same issue with building figma-linux from source as well. What version of electron-rebuild is there in your package.json?
